libs/gamify/data/prisma/schema.prisma

```diff
@@ -26,4 +26,17 @@ model Application {
     externalApplicationUrl String?
     ownerUser              User     @relation(fields: [ownerUserId], references: [id])
     ownerUserId            Int
+    Badge                  Badge[]
+}
+
+model Badge {
+    id                   Int         @id @default(autoincrement())
+    createdAt            DateTime    @default(now())
+    name                 String
+    application          Application @relation(fields: [applicationId], references: [id])
+    applicationId        Int
```

Comment:
I'm not sure what `application` would exactly refer to, but would `project` also have a similar meaning in this scenario?

Reply:
This defines the relationship of `applicationId` to the `Application` model, if that is what you mean? Don't know what you exactly mean by `project`